api/v3/views/country.rabl

```diff
@@ -43,19 +43,20 @@ end
 # Relations
 if @current_user.access_to?(Country, :country_statistic)
   child :country_statistic => :statistics do
-    attributes :pa_area, :percentage_cover_pas, :eez_area,
-      :ts_area, :pa_land_area, :pa_marine_area, :percentage_pa_land_cover,
-      :percentage_pa_eez_cover, :percentage_pa_ts_cover, :land_area, :percentage_pa_cover,
-      :pa_eez_area, :pa_ts_area, :percentage_pa_marine_cover, :marine_area,
+    attributes :pa_land_area, :pa_marine_area,
+      :land_area, :percentage_pa_land_cover,
+      :percentage_pa_marine_cover, :marine_area,
       :polygons_count, :points_count
   end
 end
 
 if @current_user.access_to?(Country, :pame_statistic)
   child :pame_statistic => :pame_statistics do
     attributes :assessments, :assessed_pas,
```

Comment:
Everything works just fine, the only two fields still returning `null` are these two.
I think Ed wanted us to check if possible to populate this in a different way:
```I propose that we check:  "assessments": 39, = COUNT ass_id per ISO3
                                             "assessed_pas": 18,= COUNT wdpa_id per ISO3 [from pame sites csv]
```
To be honest, I'm not entirely sure what he means by `ass_id` and `wdpa_id per ISO3 [from pame sites csv]` maybe it is worth it to double check with him.

Reply:
Thanks @lucacug !
It seems that those fields are not present in the provided CSV file, which explains why those values are currently empty in the first place; those were supposed to be populated with the monthly importer.

It would be good to follow up on this.
In the meantime I would need to change the importer slightly so that those fields can get populated automatically if empty. However, I guess we would then need to make sure the resulting figures are as expected.
@stacytalbot FYI